### DISCUSSION TITLE
Plugin download blocked

### DISCUSSION THREAD

**Discussion by fgfernando1945-bit** [gephi/gephi#2538] [Q&A]
Can you help me? Why when I try to download a plugin, it is reported that the download is not safe and is blocked?
![Gephi help](https://user-images.githubusercontent.com/105825221/170528454-32a86d8e-3df7-4184-9ab0-ad55b100c987.png)

**Comment by mbastian** [comment]
Do you mean when you download them directly from Gephi? You screenshot doesn't help at the moment.
